packages/ui/src/Toggle.module.scss

```diff
@@ -0,0 +1,138 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+$h: c.$grid * 6;
+$w: c.$grid * 15;
+$labelMargin: c.$grid * 4;
+$trackBr: 22px;
+$trackBgOn: #1dbc22;
+$trackBgOff: c.$colorNeutralLight;
+$knobRight: $w * 0.6;
+$knobBgOn: c.$colorNeutralWhite;
+$knobBgOff: c.$colorNeutralLighter;
+$knobR: $h * 0.9;
+$knobOffset: c.$grid * 0.3;
```

Comment:
In figma it is 2px. Our `$grid` is `0.25rem` which is an equivalent of `4px`. Should it be `0.5 * $grid`?

Reply:
nope, .5 looks way off. .3 is the magic number.

Reply:
I see. Because of the border that takes `1px`, the offset is `1px` not `2px`. Should it be `c.$grid * 0.25` then? Otherwise. it is calculated as `1.2px`.